### ISSUE TITLE
Client crash when adding bookmark to NEI

### ISSUE THREAD

**Issue by Chezzik** [GT-IMPACT/IMPACT-1#237]
Какую версию модпака вы используете? / Which modpack version are you using? 
IMPACT 1.0.1.4

Вы играете в одиночке или на сервере? / Are you playing single or on a multiplayer?
multiplayer

Опишите ошибку (Если нужно, приложите скриншоты). / Describe the bug (Attach screenshots if needed).
I was looking at the recipe for the wrought iron chest, and added it as an NEI bookmark.  The client crashed.

Какие изменения вы предлагаете? / What changes do you suggest?
Fix the crash

====
more questions
====

Repeatable? Yes, 100% of the time.

Crash log? Yes: https://gist.github.com/Chezzik/272aa1cbb2ccfcffe624fa2d3f77ab05

EDIT: It also crashes when I try to make this item.  I added the crash log in with the other one.

**Comment by Chezzik** [comment]
I tried this in single player, and there are no problems.

The chests look very different in single player than in multiplayer.  In multiplayer, the drawing of the item is wrong.  The mod for the item says "none" instead of "Chest up", and the tag is "none" instead of "chestup:Blockchestup"

Both the client and the server are 1.0.1.4 without any modifications.

**Comment by mitchej123** [comment]
I rewrote the bookmark handling in 2.0.10-GTNH   I'd give that version (or one of the beta versions) a try and see if it fixes that.  If not please report it on the GTNH repo unless impact has a custom fork.

**Comment by aagrishankov** [comment]
> I rewrote the bookmark handling in 2.0.10-GTNH I'd give that version (or one of the beta versions) a try and see if it fixes that. If not please report it on the GTNH repo unless impact has a custom fork.

we have a very old version, we will have to try your new one

**Comment by mitchej123** [comment]
2.0.11 is the latest (stableish) one before the JEI style tabs.  That's getting close to stable (beta9 right now), but still has a few kinks to work out.  If you have any new handlers you'd like to add support for their tabs hit me up on discord.

**Comment by Chezzik** [comment]
I think we figured out the problem here has nothing to do with NEI.  There was discussion in Discord that isn't reflected here.

The problems were:

1. ChestUp has a non-ASCII character in the file name (for the .jar file)
2. My server software (Pterodactyl) apparently has problems with non-ASCII characters in file names.  I couldn't even delete or rename the file.

As a result, ChestUp didn't load properly.  The bookmark issue was just the first thing I noticed, but basically everything in the mod was broken.    

I suppose it may still be worth putting in a bit more protection so that the client doesn't crash at that point, but it's probably pretty low priority.  If a player is in a similar situation as us, the client will crash when they try to craft an item like this, and you won't be able to prevent that.
